modules/ossm-threescale-authentication.adoc

```diff
@@ -90,21 +89,21 @@ You can manipulate this object using the methods described previously. In the ex
 
 [source,yaml]
 ----
-  apiVersion: "config.istio.io/v1alpha2"
-  kind: instance
-  metadata:
-    name: threescale-authorization
-  spec:
-    template: threescale-authorization
-    params:
-      Subject:
-  properties:
-          app_key: request.query_params["app_key"] | request.headers["app-key"] | ""
-          client_id: request.auth.claims["azp"] | ""
-      action:
-        path: request.url_path
-        method: request.method | "get"
-          service: destination.labels["service-mesh.3scale.net/service-id"] | ""
+apiVersion: "config.istio.io/v1alpha2"
+kind: instance
+metadata:
+  name: threescale-authorization
+spec:
+  template: threescale-authorization
+  params:
+    subject:
+    properties:
+      app_key: request.query_params["app_key"] | request.headers["app-key"] | ""
+      client_id: request.auth.claims["azp"] | ""
```

Comment:
These are under `subject`, so they require another level of indentation. /cc @dfennessy @pehala

Reply:
@unleashed updated